onnx/defs/pnp/def.cc

```diff
@@ -0,0 +1,75 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <algorithm>
+#include <cmath>
+#include <numeric>
+
+#include "onnx/defs/schema.h"
+
+namespace ONNX_NAMESPACE {
+
+ONNX_PNP_OPERATOR_SET_SCHEMA(LinalgSVD, 1,

```

Comment:
We could just call it "SVD". Especially if we change the domain to "ai.onnx.linalg" as discussed elsewhere.